src/main/java/bot/TaskList.java

```diff
@@ -0,0 +1,93 @@
+package bot;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import tasks.Task;
+
+/**
+ * Encapsulates bot task list
+ */
+public class TaskList {
+
+    private List<Task> taskList;
+    private final int MAX_TASKS = 100;
+
+    public TaskList(List<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Get the taskList
+     *
+     * @return Bot's list of tasks
+     */
+    public List<Task> get() {
+        return this.taskList;
+    }
+
+    /**
+     * Set the taskList
+     *
+     * @param taskList list of tasks
```

Comment:
I think that it would be better to avoid the setters as it would reduce the point of having a private tasklist. 
I personally prefer to only allow the private attributes be modified only via the public methods provided by the wrapper class. For example in this case I only allow the tasklist to be modified by the add and deleteTask functions, whereas the getter will create a new arraylist and copy the private tasklist over, so as to return this copy instead of the actual tasklist. 
This is prevent others from modifying the tasklist at will, which could lead to runtime exceptions

Reply:
Also a good point, was definitely being lazy with setters in various places. Some 'tell-don't-ask' seems due!